DEBUG 2021-06-04 05:31:39.327 UTC: 	 CRAB Client version: v3.210601
DEBUG 2021-06-04 05:31:39.327 UTC: 	 Running on: Linux lxplus730.cern.ch 3.10.0-1160.25.1.el7.x86_64 #1 SMP Wed Apr 28 21:49:45 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2021-06-04 05:31:39.327 UTC: 	 Executing command: 'submit'
DEBUG 2021-06-04 05:31:39.327 UTC: 	 Found CRAB cache file /afs/cern.ch/user/y/yian/.crab3
ERROR 2021-06-04 05:31:39.328 UTC: 	 CRAB configuration file crabcode_2018/EGamma_Run2018C_cfg.py not found.
ERROR 2021-06-04 05:31:39.328 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210601.00/bin/crab", line 151, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210601.00/bin/crab", line 136, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210601.00/lib/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210601.00/lib/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210601.00/lib/CRABClient/Commands/SubCommand.py", line 52, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabcode_2018/EGamma_Run2018C_cfg.py not found.
INFO 2021-06-04 05:31:39.331 UTC: 	 Log file is /afs/cern.ch/work/y/yian/work/GZ/CMSSW_10_6_20/src/PhysicsTools/NanoAODTools/WWG/2018/crab/crab2018/crab.log
